src/platform/graphics/webgpu/webgpu-texture.js

```diff
@@ -285,6 +284,12 @@ class WebgpuTexture {
                 }
             }
 
+            // ensure anisotropy is only used only when filtering is correctly
+            // set up
+            descr.maxAnisotropy = (descr.minFilter === 'linear' && descr.magFilter === 'linear') ?
```

Comment:
based on the spec, mip filter needs to be linear too

![image](https://github.com/playcanvas/engine/assets/59932779/04834f82-4ee2-4ff2-9269-1978392f75bf)

Reply:
indeed - thanks!